import-service/src/libs/s3.ts

```diff
@@ -0,0 +1,63 @@
+import AWS from 'aws-sdk';
+
+class S3 {
+    private s3;
+    private bucketName: string;
+    constructor(bucketName: string, region: string) {
+       this.bucketName = bucketName;
+       this.s3 = new AWS.S3({
+           region,
+           signatureVersion: "v4"
+       })
+    }
+
+    listObjects = async (prefix: string) => {
+        return await this.s3.listObjectsV2({
+            Bucket: this.bucketName,
+            Prefix: prefix
+        }).promise();
+    }
+
+    public copyObject = async (fileKey: string, destinationKey: string): Promise <any> => {
```

Comment:
nit: some methods are marked as public, but I don't see any `private` keywords
it would be better if you used `private` access modifier as well for better typing